### DISCUSSION TITLE
Small change to Lockpicking skill leveling gap between 170-175

### DISCUSSION THREAD

**Discussion by SimonSugoi** [chromiecraft/chromiecraft#6581] [Suggestions]
Leveling 1-170 is fairly easy as there are insta-respawning chests in Redridge and then you get a ton of junkboxes (https://www.wowhead.com/wotlk/item=16883/worn-junkbox) from pick pocketing mobs but then we hit a wall where those junkboxes turn gray at 170 and the next tier of junkboxes https://www.wowhead.com/wotlk/item=16884/sturdy-junkbox) from pick pocketing require a 175 skill. Now the only way to lvl from 170 to 175 (besides the less than 1% droprate of 125 skill junkboxes (https://www.wowhead.com/wotlk/item=4636/strong-iron-lockbox) that cannot be pickpocketed but dropped from lvl 35-ish mobs) is the dwarf keep in Badlands  where you only have 4 locked chests on the second floor but the problem is that the respawn takes literally hours. I spent a whole days camping them just to get 5 points in lockpicking. So my suggestions is to either shorten the respawn time or add more of those (https://www.wowhead.com/wotlk/object=179490/battered-footlocker) somwhere. Thank You and long live CC, the best PS there is!

**Comment by Apathyxia** [comment]
Not to take away from the harsh 170-175 normally, but you could wear [Dark Leather Gloves](https://www.wowhead.com/wotlk/item=4248/dark-leather-gloves) which increases your lockpicking skill by 5. This would let you stop at 170 and already move to 175 junkboxes.

**Reply by SimonSugoi** [reply]
Oh that's pretty clever, thank you for that, I didn't know!

**Comment by krutoj87** [comment]
I would suggest to just reduce the respawn timer of all the lockers in the world to 5 minutes. This might not be blizzlike, but it shouldn't break the game either. The lockers don't contain outrageous stuff, so you can't abuse it to farm gold. But it would improve the experience of leveling lock picking skill A LOT!

I mean it was like this in WoW Classic and it worked. And for some reason the lockers in the water around sunken temple also have a short respawn of a few minutes. Why shouldn't this be applied to other lockers aswell?

And there are ways to level lock picking without relying on lockers. But they all require game knowledge, that new players or players, that never played a rogue before won't have.
